api/panda_studio.yaml

```diff
@@ -250,3 +365,58 @@ components:
         file:
           type: string
           format: binary
+
+    Recording:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        description:
+          type: string
+        imageId:
+          type: string
+        recordingFiles:
+          type: array
+          items:
+            $ref: "#/components/schemas/RecordingFile"
+        config:
+          $ref: "#/components/schemas/PandaConfig"
+
+    RecordingFile:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        file_type:
+          $ref: "#/components/schemas/ImageFileType"
```

Comment:
Not applicable to recording

Reply:
Or at the least, this should be changed to RecordingFileType with possible values "snp" and "ndlog"